Jenkinsfile

```diff
@@ -33,8 +33,8 @@ timestampedNode('java') {
     timeout(time: 180, unit: 'MINUTES') {
         // See below for what this method does - we're passing an arbitrary environment
         // variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
-        withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m",
-                      "MAVEN_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m"]) {
+        withMavenEnv(["JAVA_OPTS=-Xmx2g -Xms512m -XX:MaxPermSize=1024m",
```

Comment:
Fair enough. I'll admit that I just blindly throw RAM at Maven. =) Would you prefer just the Surefire change, then? I'd like to leave the overall Xmx as it was in that case, though, rather than lowering it until we've gotten things stable.